app/cypress/e2e/1-ndr-smoke-tests/gp_user_workflows/patient_search_and_verify_unhappy_workflow.cy.js

```diff
@@ -0,0 +1,34 @@
+import { Roles, roleName } from '../../../support/roles';
+
+describe('GP Workflow: Patient search unhappy path', () => {
+    // env vars
+    const baseUrl = Cypress.config('baseUrl');
+    const gpRoles = [Roles.GP_ADMIN_H85686, Roles.GP_CLINICAL_H85686];
+
+    const workspace = Cypress.env('WORKSPACE');
+    const activePatient = workspace === 'ndr-dev' ? '9730153817' : '9000000002';
+    const homeUrl = '/home';
+
+    gpRoles.forEach((role) => {
+        it(
+            `[Smoke] Show that patient is not accessable for this  ${roleName(role)} `,
```

Comment:
I think the title could be a bit more descriptive, as I am not 100% clear what this test is proving. 

Is the patient inaccessible because the ODS code of the user is different to that of the patient? In which case, a better test name could be 'it does not allow the user to access the record of a patient registered at a different organisation'.